libcxx/include/__compare/three_way_comparable.h

```diff
@@ -37,6 +38,20 @@ concept three_way_comparable =
       { __a <=> __b } -> __compares_as<_Cat>;
     };
 
+#  if _LIBCPP_STD_VER >= 23
+
+template <class _Tp, class _Up, class _cat = partial_ordering>
```

Comment:
```suggestion
template <class _Tp, class _Up, class _Cat = partial_ordering>
```
I believe we use capital letter for template parameters.

Reply:
Yep, sorry. Fixed.